package.json

```diff
@@ -9,6 +9,8 @@
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
     "object.fromentries": "^2.0.2",
+    "socket.io": "^2.3.0",
+    "socket.io-client": "^2.3.0",
```

Comment:
What is the client used for in the server code?

Reply:
Removed. In adding socket.io to the client side, I added this and forgot to remove it when I didn't need it.